test/unbox.test.js

```diff
@@ -36,8 +36,10 @@ test('unbox', async t => {
   }
 
   const RECPS = [
+    // TODO: i think i asked this somewhere else, but do we actually wanna support DMs in the first slot in this module? what's the usecase?
     ssb.id,
     groupId,
+    // TODO: p.s. do we have some test that tests DMing a pobox actually? that's a non-groupId that should be allowed in the first slot, i guess
```

Comment:
maybe good to have, maybe ok to do in a separate issue/pr though

Reply:
made issue https://github.com/ssbc/ssb-tribes/issues/99

Reply:
`recps: [poBoxId, feedId]`